lib/debug/session.rb

```diff
@@ -107,6 +107,11 @@ def reset_ui ui
       @management_threads << @ui.reader_thread if @ui.respond_to? :reader_thread
     end
 
+    def disable_session_tps
+      @tp_load_script.disable
+      @tp_thread_begin.disable
```

Comment:
I haven't found any issue caused by this TP yet. But I think it makes sense to disable it too.